src/Begin.f90

```diff
@@ -13,10 +13,10 @@ module BGIN
     character (len=10):: long
 !
 ! Integer variables
-!
+!
```

Comment:
@jyearsley : Try to avoid whitespace only changes. They end up cluttering code review